docs/streaming/apis-on-dataframes-and-datasets.md

```diff
@@ -436,7 +437,7 @@ Imagine our [quick example](./getting-started.html#quick-example) is modified an
 
 The result tables would look something like the following.
 
-![Window Operations](/img/structured-streaming-window.png)
+![Window Operations]({{ rel_path_to_root }}img/structured-streaming-window.png)
```

Comment:
It looks unfriendly for markdown writers

Reply:
I had originally believed that there was no cleaner way to do this, but let me lay out an alternative. The main trickiness is that we pre-build HTML pages to put under `/docs/<version>/`, so we cannot:

1. Use absolute paths, since those resolve under spark.apache.org, not spark.apache.org/docs/<version>.
2. Use sibling paths, since doing `img/foo.png` on a `docs/<version>/streaming/` page will resolve to `/streaming/img/`, and we have our images under `/docs/<version>/img`.

However, one option is to actually move all streaming related images under `/docs/<version>/img`. Then, all image usages in streaming doesn't need any of this `rel_path_to_root` stuff. I originally didn't do this because I wasn't sure if we had some special process for serving the content in `/img` from a CDN, but it seems like it's being served by an Apache web server (though I can't tell more from the Devtools).

Does that sound better to you?

Reply:
@yaooqinn, bump on this!

Reply:
AFAIK, a relative path can be visited by symbols like '.', '..', which is supported by most of documentation generators.

Reply:
Oh, yeah, you're totally right. Since the stuff in `global.html` _does_ need the `rel_path_to_root` stuff, I was thinking I needed it in other places. But I don't. Relative paths will work just fine for streaming-only pages.

Reply:
Can you attach the screenshot after this PR？